### ISSUE TITLE
#Hacktoberfest

### ISSUE THREAD

**Issue by arguiot** [arguiot/blogy#2]
Hacktoberfest 2016

**Comment by owenvoke** [comment]
Does something need to be done for this? 😄

**Comment by lucasgruwez** [comment]
Nah, he's just hacking and drinking beer in his bed... Or he wants to win a [Hacktoberfest](https://hacktoberfest.digitalocean.com/) T-shirt
